packages/designmanual/stories/pages/SearchResultTypeExample.jsx

```diff
@@ -318,6 +318,7 @@ const SearchPageDemo = ({ t }) => {
         dispatchResources({ type: 'RESOURCE_TYPE_SELECTED', ...data });
       }
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
hva er det du disabler når du disabler react-hooks/exhaustive-deps? Fra et raskt google søk ser det ut som om det ikke er best practice

Reply:
disabler følgende warning:
warning  React Hook useEffect has a missing dependency: 'resourceItems'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

Etter oppgradering av typescript så var det en del slike som dukket opp. Har fikset de fleste, men det er et par i designmanulen som krever noe omskriving, derfor disablet jeg de. Siden de er i designmanualen og ikke i pakkene så tenker jeg at det ikke er så hast med å fikse